pkg/gui/controllers/helpers/refs_helper.go

```diff
@@ -27,19 +28,22 @@ type RefsHelper struct {
 	git      *commands.GitCommand
 	contexts *context.ContextTree
 	model    *types.Model
+	config   *config.UserConfig
 }
 
 func NewRefsHelper(
 	c *types.HelperCommon,
 	git *commands.GitCommand,
 	contexts *context.ContextTree,
 	model *types.Model,
+	config *config.UserConfig,
```

Comment:
no need to pass this in because the config already lives in HelperCommon, so you can use `self.c.UserConfig`